graph_weather/data/const.py

```diff
@@ -5638,3 +5640,475 @@
     "WEASD.surface": -12.248001098632812,
     "WILT.surface": -0.004800006747245789,
 }
+
+
+ERA5_MEANS = {
```

Comment:
do you mind add a comment why this is a 1-D array?

Reply:
Done!

Reply:
Thanks, this is perfect